### ISSUE TITLE
Doesn't work for TaoTronics headset

### ISSUE THREAD

**Issue by tomtastic** [jariz/Akku#28]
MacOS : Mojave 10.14.6
Akku : Release 0.1.0-beta.8
Headset : TaoTronics TT-BH026

```
$ pgrep -fli akku
578 /Applications/Akku.app/Contents/MacOS/Akku
579 /Library/PrivilegedHelperTools/io.jari.AkkuHelper
```
`$ system_profiler SPBluetoothDataType`
```
          TaoTronics TT-BH026:
              Address: 00-02-5B-13-D3-FB
              Major Type: Audio
              Minor Type: Headset
              Services: Hands-Free unit, Hands-Free unit, Headset
              Paired: Yes
              Configured: Yes
              Connected: Yes
              Manufacturer: Cambridge Silicon Radio (0x7, 0x21C8)
              Firmware Version: 0x0000
              Class of Device: 0x04 0x01 0x240404
              RSSI: -53
              Role: Master
              Connection Mode: Sniff Mode
              Interval: 500 ms
              EDR Supported: Yes
              eSCO Supported: Yes
              SSP Supported: Yes
```
![IMG_645743695009-1](https://user-images.githubusercontent.com/302127/62009960-b5665180-b15c-11e9-8021-b5a35a513027.jpeg)
![Untitled](https://user-images.githubusercontent.com/302127/62010021-5bb25700-b15d-11e9-9fdb-be00f2729747.png)

Tried disconnect, reconnecting the headset multiple times, but Akku seems unable to find the battery level details.

**Comment by jariz** [comment]
Hi, thanks for checking out the project.  It's in pretty early stages so I'll need to test it with some more hardware (like yours!) before it becomes really useful.  

So, I require a bit more data to fully understand the problem.  
I just set up a guide [here](https://github.com/jariz/Akku/blob/master/CONTRIBUTING.md) that you can follow!

**Comment by tomtastic** [comment]
Bluetooth Reporter Dump + '~/Library/logs/Akku.log'
https://we.tl/t-pLJsBtZuQZ

Akku.log is pretty incriminating though!
```
29-07-2019 17:37:47 ERROR helper():176 - Helper connection was closed with error: Error Domain=NSCocoaErrorDomain Code=4097 "connection to service named io.jari.AkkuHelper" UserInfo={NSDebugDescription=connection to service named io.jari.AkkuHelper}
```

**Comment by jariz** [comment]
@tomtastic Just released a new update that should likely fix this. Can you give it a try?

**Comment by tomtastic** [comment]
Hi, I installed the latest version, and was prompted to install the helper, though after clicking the (then highlighted and active looking) 'Install my helper app' button, nothing appears to have happened, I wasn't prompted for admin credentials and now the popup just has a greyed out button.
![helper](https://user-images.githubusercontent.com/302127/62937069-5ee74d00-bdc3-11e9-808d-4b5eadd21c66.jpg)

**Comment by tomtastic** [comment]
I killed the Akku and io.jari.AkkuHelper processes and restarted the app, after which it detects the headset but no battery information could be found until I disconnect and reconnect the bluetooth device at which point it does pickup the battery status.

NB: This is actually for my Sony WH-1000MX3 headphones, but they exhibited the same problem as the Taotronics previously and they're all I have to had until this evening.

```
13-08-2019 12:18:25 DEBUG 80():127 - BluetoothDeviceAddress(data: (112, 38, 5, 182, 87, 114))
13-08-2019 12:18:30 DEBUG 80():127 - BluetoothDeviceAddress(data: (112, 38, 5, 182, 87, 114))
13-08-2019 12:18:35 DEBUG 80():127 - BluetoothDeviceAddress(data: (112, 38, 5, 182, 87, 114))
13-08-2019 12:18:40 DEBUG 80():127 - BluetoothDeviceAddress(data: (112, 38, 5, 182, 87, 114))
13-08-2019 12:18:45 DEBUG 80():127 - BluetoothDeviceAddress(data: (112, 38, 5, 182, 87, 114))
13-08-2019 12:18:50 DEBUG 80():127 - BluetoothDeviceAddress(data: (112, 38, 5, 182, 87, 114))
13-08-2019 12:18:55 DEBUG 80():127 - BluetoothDeviceAddress(data: (112, 38, 5, 182, 87, 114))
13-08-2019 12:19:00 DEBUG 80():127 - BluetoothDeviceAddress(data: (112, 38, 5, 182, 87, 114))
13-08-2019 12:19:05 DEBUG 80():127 - BluetoothDeviceAddress(data: (112, 38, 5, 182, 87, 114))
13-08-2019 12:19:10 DEBUG 80():127 - BluetoothDeviceAddress(data: (112, 38, 5, 182, 87, 114))
```

**Comment by tomtastic** [comment]
I just tested my TaoTronics TT-BH026 and Akku is now working, though only after killing and restarting the app, no amount of disconnect+reconnecting prior to this would detect the battery life.

Happy to close this ticket now, thank you!

**Comment by jariz** [comment]
Yes, the issue with app hanging during helper install is something I've seen reported before. I'll create a separate issue for that.
Glad it's working for you!
